homeassistant/components/goalzero/strings.json

```diff
@@ -16,7 +20,7 @@
       "unknown": "[%key:common::config_flow::error::unknown%]"
     },
     "abort": {
-      "already_configured": "[%key:common::config_flow::abort::already_configured_account%]"
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
```

Comment:
`invalid_host` and `unknown` need to be added here

Reply:
Already included in another PR but will change it here.